include/fmt/chrono.h

```diff
@@ -408,6 +460,9 @@ struct formatter<std::chrono::duration<Rep, Period>, Char> {
     begin = internal::parse_align(begin, end, handler);
     if (begin == end) return begin;
     begin = internal::parse_width(begin, end, handler);
+    if (begin == end) return begin;
+    begin = parse_precision(begin, end, handler, std::is_floating_point<Rep>{});
```

Comment:
Do we actually need tag dispatched `parse_precision`? Wouldn't `if (std::is_floating_point<Rep>{})` work and be much simpler?

Reply:
We don't need it. The tag dispatch is just a clear hint to the compiler to not even try instantiating dead code.